src/lib/gl_engine/tvgGlGpuBuffer.cpp

```diff
@@ -29,7 +29,7 @@
 GlGpuBuffer::GlGpuBuffer()
 {
     GL_CHECK(glGenBuffers(1, &mGlBufferId));
-    assert(mGlBufferId != GL_INVALID_VALUE);
```

Comment:
doesn't GL_INVALID_VALUE zero? it is more obvious to use.

Reply:
`GL_INVALID_VALUE ` is a [OpenGL error code](https://www.khronos.org/opengl/wiki/OpenGL_Error),  and the value is 0x501.
The buffer id may reach this value if glGenBuffers is called multiple times, but it is a valid buffer id

Reply:
gotcha! thanks.